Tensile/Source/lib/include/Tensile/ContractionSolution.hpp

```diff
@@ -148,18 +152,22 @@ namespace Tensile
         /**
    * Generate a set of kernel calls to solve a particular problem.
    */
-        virtual std::vector<KernelInvocation>
-            solve(Problem const& problem, Inputs const& inputs, Hardware const& hardware) const;
+        virtual std::vector<KernelInvocation> solve(Problem const&           problem,
```

Comment:
Adding arguments here needs to be very carefully considered, as this is part of the main interface to Tensile.  We can't add any 'debug' type arguments here.